### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by ProgramKingAbel**
# View Wireframes :eyes: 

- [x] Implement the design wireframes/Viewa with preprocessed HTML and erb code
- [x]  add seed data to database for testing the View Renders
- [x]  Used layouts and templates for shared content.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,3 +1,13 @@
-<h1> Displays each users Details </h1>
 
-<p>Name: <strong><%= @user.name %> id: <%=@user.id %></strong></p>
+  <%= render partial: 'partials/user', locals: { user: @user } %>
+  <div class="bio">
+    <h3>Bio</h3>
+    <p><%=@user.bio%></p>
+  </div>
+  <div class="posts">
+    <% @user.posts.limit(3).each do |post| %>
```
Comment (anagudelogu) [inline app/views/users/show.html.erb]: - [ ] Please consider using the method you created in an earlier milestone: 🤓 🔧 

https://github.com/ProgramKingAbel/tubongeApp/blob/eb0868d9b78f13de205bfbab241cd64ec1b4b71d/app/models/user.rb#L9-L11

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,11 @@
-<h1> Implement all posts for associated to a specific user</h1>
+  <%= render partial: 'partials/user', locals: {user: @user} %>
+   <% @user.posts.each do |post| %>
+     <%= render partial: 'partials/post', locals: { post: post } %>
+     <div class="all-comments">
+    <% post.comments.limit(5).each do |comment| %>
```
Comment (anagudelogu) [inline app/views/posts/index.html.erb]: - [ ] Please consider using the method you created earlier: 🤓 🔧 

https://github.com/ProgramKingAbel/tubongeApp/blob/eb0868d9b78f13de205bfbab241cd64ec1b4b71d/app/models/post.rb#L16-L18

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @ProgramKingAbel, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @ProgramKingAbel,

✔️ Views match wireframe

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**